Put VS 2015 redistributable files into deployment package for Windows

If user doesn't have installed **Visual Studio 2015** redistributable then the game won't start and shows error message with missing **MSVCP140.dll** library. We have to include this file into deployment package for x32 and x64 versions.

Alternatively, we could try to compile the application with `/MTd` flag for debug mode and `/MT` for release mode.

my VS2015 setup says this during the build, BTW:

```
1>LINK : warning LNK4098: defaultlib 'MSVCRT' conflicts with use of other libs; use /NODEFAULTLIB:library
```

seems like `SDL.dll` and `SDL_Mixer.dll` both linked with another version of MSVCRT.dll:

![depends_sdl](https://user-images.githubusercontent.com/7666511/67486800-5c338e80-f675-11e9-8fd4-7812b767299a.png)
![depends_sdl_mixer](https://user-images.githubusercontent.com/7666511/67486842-6fdef500-f675-11e9-9ae3-f0c67e291d4a.png)

May be these dlls should be rebuilt statically or with the same version of MSVCRT on their own.

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -46,9 +46,9 @@ install:
   - cd C:\projects\fheroes2
 
 after_build:
+  - cmd: cd C:\projects\fheroes2\build\%platform%\%configuration%
   - cmd: if "%platform%" == "x86" xcopy /Y /s /Q "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.CRT\msvcp140.dll" "."
   - cmd: if "%platform%" == "x86" xcopy /Y /s /Q "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.CRT\vcruntime140.dll" "."
-  - cmd: cd C:\projects\fheroes2\build\%platform%\%configuration%
   - cmd: 7z a fheroes2_windows_%platform%_%deploy_conf_name%.zip fheroes2.exe SDL*.dll lib*.dll zlib*.dll msvcp*.dll vcruntime*.dll
   - cmd: cd C:\projects\fheroes2
   - cmd: xcopy /Y /s /Q "doc\README.txt" "."

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -46,6 +46,8 @@ install:
   - cd C:\projects\fheroes2
 
 after_build:
+  - cmd: if "%platform%" == "x86" xcopy /Y /s /Q "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.CRT\msvcp140.dll" "."
+  - cmd: if "%platform%" == "x86" xcopy /Y /s /Q "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.CRT\vcruntime140.dll" "."
   - cmd: cd C:\projects\fheroes2\build\%platform%\%configuration%
   - cmd: 7z a fheroes2_windows_%platform%_%deploy_conf_name%.zip fheroes2.exe SDL*.dll lib*.dll zlib*.dll
   - cmd: cd C:\projects\fheroes2

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -49,7 +49,7 @@ after_build:
   - cmd: if "%platform%" == "x86" xcopy /Y /s /Q "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.CRT\msvcp140.dll" "."
   - cmd: if "%platform%" == "x86" xcopy /Y /s /Q "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\redist\x86\Microsoft.VC140.CRT\vcruntime140.dll" "."
   - cmd: cd C:\projects\fheroes2\build\%platform%\%configuration%
-  - cmd: 7z a fheroes2_windows_%platform%_%deploy_conf_name%.zip fheroes2.exe SDL*.dll lib*.dll zlib*.dll
+  - cmd: 7z a fheroes2_windows_%platform%_%deploy_conf_name%.zip fheroes2.exe SDL*.dll lib*.dll zlib*.dll msvcp*.dll vcruntime*.dll
   - cmd: cd C:\projects\fheroes2
   - cmd: xcopy /Y /s /Q "doc\README.txt" "."
   - cmd: 7z a build\%platform%\%configuration%\fheroes2_windows_%platform%_%deploy_conf_name%.zip script\demo\demo_windows.bat LICENSE fheroes2.key changelog.txt README.txt

